src/main/java/org/jenkinsci/test/acceptance/po/Job.java

```diff
@@ -278,7 +278,13 @@ public Build getLastBuild() {
     public <T extends Parameter> T addParameter(Class<T> type) {
         ensureConfigPage();
 
-        check(find(by.xpath("//input[@name='parameterized']")));
```

Comment:
I suppose we could have a PR builder for this repo that runs all tests against both LTS and trunk Jenkins code. @andresrc might volunteer…?